spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+  describe "games#create action" do
+    it "should successfully create new game in the database" do
+      post :create, params: { game: {game_id: 1, user_white_id: 1, user_black_id: 2} }
```

Comment:
When creating a new game, Rails will autoassign the `id`, so you don't need to pass `game_id: 1` here.